include/motor-control/core/stepper_motor/motion_controller.hpp

```diff
@@ -73,7 +73,7 @@ class MotionController {
             .seq_id = can_msg.seq_id,
             .stop_condition = can_msg.request_stop_condition,
             .usage_key = hardware.get_usage_eeprom_config().get_distance_key()};
-        if (!enabled) {
+        if (!enabled && velocity_steps != 0) {
```

Comment:
Would also want to check `acceleration_steps != 0`, otherwise the motor won't get enabled for a move step that accelerates from 0

Reply:
good call!